webpack.config.js

```diff
@@ -25,14 +28,14 @@ const copy_files = [
 const plugins = [
     new copy({ patterns: copy_files }),
     new extract({filename: "[name].css"}),
-    new ESLintPlugin({
-        extensions: ["js", "jsx"],
-        failOnWarning: true,
-    }),
     new CockpitPoPlugin(),
     new CockpitRsyncPlugin({dest: packageJson.name}),
 ];
 
+if (eslint) {
+    plugins.push(new ESLintPlugin({ extensions: ["js", "jsx"], failOnWarning: true, }));
+}
+
```

Comment:
hmm why are these changes here? Ideally this should go through starterkit first right?

Reply:
well, it is already in cockpit anyway :) We can add it into starter kit, but it seems like pattern that you don't want people to use, it is just good hack when you quickly iterate over the code